server/views/review/includes/booleanYesNo.pug

```diff
@@ -0,0 +1,8 @@
+mixin booleanYesNo(id, property)
+  span(id=id).block.bold.midMarginBottom
+    if property === true
+      | Yes
+    else if property === false
+      | No
+    else
+      | -
```

Comment:
NB the purpose of this is to prevent 'undefined' being treated as false
The value will have been set explicitly to false when user input means false, but will be undefined when not yet answered